runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -1299,12 +1299,6 @@ ROMClassBuilder::computeExtraModifiers(ClassFileOracle *classFileOracle, ROMClas
 		}
 	}
 
-	if ( classFileOracle->isSynthetic() ) {
-		/* handle the synthetic attribute. In java 1.5 synthetic may be specified in the access flags as well so do not unset bit here */
-		// Trc_BCU_createRomClassEndian_Synthetic(romClass);
-		modifiers |= J9AccSynthetic;
```

Comment:
Will this cause the below test to fail ?

https://github.com/eclipse-openj9/openj9/blob/7243f7a4e79b039c28fc469fe64557ec46230bad/runtime/bcutil/test/dyntest/romclass_correctness.c#L108-L110

There is another place checking if J9AccSynthetic is set below in `romclass_correctness.c`

Reply:
The existing `J9ROMCLASS_IS_SYNTHETIC()` can also be removed.

Reply:
I think it should be fine, this looks for the synthetic flag in rom class `modifiers` which isn't effected by this change.

Where does this test code run? I ran a personal build with this change and all sanity.functional tests are passing.

Reply:
> this looks for the synthetic flag in rom class modifiers which isn't effected by this change

Right, the macro is for `romClass->modifiers`. We should be fine.